### PR TITLE
colors for both debit and credit are defined for transaction history

### PR DESCRIPTION
**Pull request by usamabutt101**
### Why

Please give a short description why these code changes are being made, by replacing this line.

Link to Jira ticket: <place here>

### ToDo

- [ ]
- [ ]
- [ ]

If this is a `Draft:` or `WIP:` PR, please describe still needed actions, else delete section

### CODE REVIEW

File: src/components/adb-transaction-history/index.tsx [hunk 1]
```diff
@@ -14,56 +14,45 @@ import {
...
-    title: "Last 30 days",
+    title: 'Last 30 days',
   },
   {
     value: 60,
-    title: "Last 60 days",
+    title: 'Last 60 days',
   },
   {
     value: 90,
```
Comment (sharmal33) [inline src/components/adb-transaction-history/index.tsx:54]: plz move all title values in to translation file

File: src/components/adb-transaction-history/index.tsx [hunk 2]
```diff
@@ -306,39 +274,36 @@ const ADBTransactionHistoryComponent = (prop: ADBTransactionHistoryProps) => {
...
-      .replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,");
+    return (Math.round(n * 100) / 100).toFixed(2).replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
   };
 
   useEffect(() => {
-    if (
-      wallets.length > 0 &&
-      wallets[0].bankAccount?.accountSubType === "SavingAccount-i"
-    ) {
+    if (wallets.length > 0 && wallets[0].bankAccount?.accountSubType === 'SavingAccount-i') {
```
Comment (sharmal33) [inline src/components/adb-transaction-history/index.tsx:299]: this logic is wrong
**wallets[0]** some time it return different account type.

File: src/components/adb-transaction-history/index.tsx [hunk 3]
```diff
@@ -680,8 +623,8 @@ const styles = StyleSheet.create({
     width: 40,
   },
   groupDateFilterOptionsContainer: {
-    flexDirection: "row",
-    flexWrap: "wrap",
+    flexDirection: 'row',
```
Comment (sharmal33) [inline src/components/adb-transaction-history/index.tsx:626]: plz move all colors into translation file

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/components/adb-transaction-history/index.tsx (hunk 2 above)
Comment (usamabutt101) [reply src/components/adb-transaction-history/index.tsx:299]: I have asked this from Viet at that time.

File: src/components/adb-transaction-history/index.tsx (hunk 3 above)
Comment (usamabutt101) [reply src/components/adb-transaction-history/index.tsx:626]: resolved!

**Review by sharmal33 — APPROVED** [review]

**Pull request merged**